code.user.js

```diff
@@ -1023,6 +1027,17 @@
         }
         return null;
     }
+
+    function isRetryMessage(message) {
+        var messageList = [
+            "You have too many listings pending confirmation. Please confirm or cancel some before attempting to list more.",
```

Comment:
I don't think that's a retry message?

Reply:
I consider a retry message to be any message that will eventually go away if we retry the listing enough times.  The other 3 messages will resolve themselves after a period of time has passed.  This message will resolve after the user clears out their pending confirmations.

Reply:
Yeah but there's no point in retrying this until the user manually confirmed things, so I don't think this should be added here. I'll make some changes and merge.